s3/src/main/mima-filters/2.0.0-M3.backwards.excludes/PR2183

```diff
@@ -1,4 +1,4 @@
-# MiMa upgrade
+# MiMa upgrade shows new error, method is private
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.alpakka.s3.S3Settings.this")
```

Comment:
That's a bit unexpected. I checked and it seems if you have a scala private constructor, it will usually also be private in bytecode. However, if there's an access to the constructor from the companion object, the constructor might be made public in bytecode, otherwise it couldn't be accessed.

Reply:
In any case, it's private from Scala, so there shouldn't be any compat issues (I don't know if we even care about those currently).